client/register-businesses.py

```diff
@@ -1,23 +1,77 @@
 #!/usr/bin/env python3
-import argparse
 import glob
 import json
 import random
-import os 
+
+import click
 
 from random_words import RandomWords
 from wonderwords import RandomSentence
 
-from py_client.arg_parser import simple_parser
-from py_client.client import Client
 from py_client.ipfs import Ipfs
-from py_client.helpers import purge_prompt, read_cid, mkdir_p
+from py_client.helpers import purge_prompt, read_cid, mkdir_p, set_local_or_remote_chain
 
 BUSINESSES_PATH = './test-data/bazaar/businesses'
 OFFERINGS_PATH = './test-data/bazaar/offerings'
 
 ICON_PATH = './test-data/icons/community_icon.png'
 
+global IPFS_LOCAL
+
+@click.command()
+@click.option('--client', default='../target/release/encointer-client-notee', help='Client binary to communicate with the chain.')
+@click.option('--port', default='9944', help='ws-port of the chain.')
+@click.option('-l', '--ipfs_local', is_flag=True, help='if set, local ipfs node is used.')
+@click.option('-r', '--remote_chain', default=None, help='choose one of the remote chains: gesell, gesell-dot, gesell-ksm, cantillon-dot, cantillon-ksm.')
+def register(client, port, ipfs_local, remote_chain):
```

Comment:
I prefer the function name to be more specific: `register_businesses`. I wonder here too. As it is the only cmd in the script, can it be omitted when running the script?

Reply:
yes, you don't need to add the command in the cli, just ./register-businesses.py [OPTIONS]